parser/parser.go

```diff
@@ -104,12 +105,19 @@ func ParseStringEntitlement(entitlementFormat string) (domain []string, id, valu
 	}
 
 	idAndArgList := strings.Split(idAndArgString, "=")
-	if len(idAndArgList) != 2 {
+	id = idAndArgList[0]
```

Comment:
non-blocking nit: this assignment should remain after the `if len(idAndArgList) > 2 {` check

Reply:
Good catch!